scripts/Config.json

```diff
@@ -8,5 +8,6 @@
         "MAXSIZE": "1000",
         "POPULARITY_OUTPUT_FILE":"scripts/Popularity/popularity_results.txt",
         "TIME_SPAN":"365",
-        "SO_TOKEN":"enter your SO token"
+        "SO_TOKEN":"enter your SO token",
+        "OUTPUT_PATH": "scripts/"
```

Comment:
When create the docker, set it to `"OUTPUT_PATH": "../home/scripts/"` The shared charts folder is in home/scripts/charts
![Screen Shot 2021-02-12 at 8 33 52 AM](https://user-images.githubusercontent.com/35275868/107788862-2012ba00-6d0e-11eb-9120-9708cfe586e6.png)